crypto/types/src/lib.rs

```diff
@@ -7,6 +7,8 @@ pub mod errors;
 /// Implements [ChainKeypair](keypairs::ChainKeypair) and [OffchainKeypair](keypairs::OffchainKeypair),
 /// the important representations of chain key and packet key.
 pub mod keypairs;
+/// Implementation of the Lioness PRP
+#[allow(deprecated)] // Until the dependency updates to newer versions of `generic-array`
```

Comment:
Would it be possible (in a separate PR) to update the generic-array? It's the single remaining deprecated dependency in our stack.